README.md

```diff
@@ -1,24 +1,48 @@
-# README
+# Junk4Dollars
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Build Status](https://travis-ci.com/riccjohn/junk4dollars-api.svg?branch=master)](https://travis-ci.com/riccjohn/junk4dollars-api)
 
-Things you may want to cover:
+# Installation
 
-* Ruby version
+## Rails
 
-* System dependencies
+Install `rails 6.0.0`
```

Comment:
Does one need to install Rails separately? Or can I just use Bundler after cloning the project?